assets/src/edit-story/panels/size.js

```diff
@@ -45,11 +44,16 @@ function SizePanel( { selectedElements, onSetProperties } ) {
 						height: typeof newWidth === 'number' && lockRatio ? newWidth / ratio : height,
 					} );
 				} }
-				postfix={ _x( 'px', 'pixels, the measurement of size', 'amp' ) }
 				disabled={ isFullbleed }
 			/>
-			<InputGroup
-				label={ __( 'Height', 'amp' ) }
+			<IconSwitch
```

Comment:
This component should have a configurable (ARIA) label for accessibility. Please follow best practices here.

Reply:
Agreed!